packages/ui/src/components/vuestic-mixins/SelectableComponent/SelectableMixin.js

```diff
@@ -73,21 +82,21 @@ export const SelectableMixin = {
         return
       }
       if (this.modelIsArray) {
-        if (!this.value) {
-          this.$emit('input', [this.c_arrayValue])
-        } else if (this.value.includes(this.c_arrayValue)) {
-          this.$emit('input', this.value.filter(option => option !== this.c_arrayValue))
+        if (!this.c_value) {
+          this.valueComputed = [this.c_arrayValue]
```

Comment:
Couple of things I want to point here and in some other places.

1) The way `valueComputed` and statefullness is intended to be used is that you don't use `c_value` at all! You always interact with `valueComputed` as well as perform checks on it.
But:
2) Array access + stateful makes no sense. So for array access we pretend `computedValue` does not exist and use `c_value` + emit input directly.

To surmise:
* for array access use `c_value`.
* otherwise use `valueComputed`